types/reactstrap/lib/DropdownMenu.d.ts

```diff
@@ -1,12 +1,15 @@
 import * as React from 'react';
 import { CSSModule } from '../index';
+import { Popper } from './Popper';
```

Comment:
I hate to be the one to day this since this was here before you got to it, but we should import the typings directly from Popper, not re-implement them locally.

Reply:
Hi @FaithForHumans , thanks for the feedback ;
But I am not sure to understand what to do, I use the same syntax that is currently used within Popover and Tooltip.
Shouldn't it be a PR on its own then?
Otherwise can you please tell me where to find those typings?

Reply:
Look at types/bootstrap for an example: [package.json](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/db303437a0a80dd02f93657b7aca631f98c4a8d6/types/bootstrap/package.json) [import](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/70d32cd228c442786ae44ddf3acb821693df629f/types/bootstrap/index.d.ts#L9)

That being said, replacing everything else that's currently using the local definitions would be fine as a separate PR, but since it looks like it's just `Tooltip` using the local `Popper` definitions, if you wanted to replace that too (should just be an import change and delete the existing `Popper.d.ts` file), that'd make things easy.

Reply:
Okay, fixed by 0304145

Reply:
Thanks, I appreciate it!